Linker error with ConsensusAlgorithm

For once I think it's not me :-) This is what I get compiling master right now:
```
lib/libdeal_II.so.9.2.0-pre: error: undefined reference to 'dealii::Utilities::MPI::ConsensusAlgorithm_NBX<unsigned int, unsigned int>::ConsensusAlgorithm_NBX(dealii::Utilities::MPI::ConsensusAlgorithmProcess<unsigned int, unsigned int>&, ompi_communicator_t* const&)'
```
Are we forgetting to instantiate something?

--- a/source/base/mpi_consensus_algorithm.cc
+++ b/source/base/mpi_consensus_algorithm.cc
@@ -24,6 +24,10 @@ namespace Utilities
   {
     template class ConsensusAlgorithmProcess<unsigned int, unsigned int>;
 
+    template class ConsensusAlgorithm_NBX<unsigned int, unsigned int>;
+
+    template class ConsensusAlgorithm_PEX<unsigned int, unsigned int>;
+
     template class ConsensusAlgorithmSelector<unsigned int, unsigned int>;
 
 
@@ -32,6 +36,14 @@ namespace Utilities
       unsigned int>;
 
     template class ConsensusAlgorithmSelector<
+      std::pair<types::global_dof_index, types::global_dof_index>,
+      unsigned int>;
+
+    template class ConsensusAlgorithm_NBX<
+      std::pair<types::global_dof_index, types::global_dof_index>,
+      unsigned int>;
+
+    template class ConsensusAlgorithm_PEX<
       std::pair<types::global_dof_index, types::global_dof_index>,
       unsigned int>;
 

